devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -147,6 +162,14 @@ def installed() {
 def updated() {
 	sendHubCommand(pause())
 	state.reverseDirection = reverseDirection ? reverseDirection : false
+    int time = calibrationTime
+    
+    if (calibrationTime >= 5 && calibrationTime <= 255) {
+        response([
+    		secure(zwave.configurationV1.configurationSet(parameterNumber: 35, size: 1, scaledConfigurationValue: time)),
+        ])
+    } else {}
```

Comment:
Please remove the empty "else" block.

Reply:
@KKlimczukS - removed else block on line 171